frontend/app/services/docs.js

```diff
@@ -0,0 +1,27 @@
+import Service, { service } from "@ember/service";
+
+import config from "timed/config/environment";
+
+const ROUTE_DOCS_MAPPING = {
+  // index.name: DocsUrl
```

Comment:
If this is a hint for the structure of this mapping do it right (e.g. `<route: URL>`) or delete this comment.